admin_manual/configuration/server/occ_command.rst

```diff
@@ -1413,6 +1414,18 @@ Remove a certificate:
 
  sudo -u www-data php occ security:remove [certificate name]
 
+.. _ransomeware_commands_label:
+
+Ransomeware
+-----------
+
+Use these commands to help protect your system against Ransomeware.
```

Comment:
This is quite inaccurate IMO. The commands do not really help _protecting_ against Ransomware (also Ransomware is misspelled). The blocker is kind of a protection but the actual commands that the app introduces can just help after an attack has occurred. So it's rather a Ransomware remedy or something like that :).

Reply:
This is only a slight modification from the original text, but thanks for pointing that out.